### PR TITLE
Add test case for file manager opening and closing

### PR DESCRIPTION
**Pull request by osamacis**
Description :
- Add test case for file manager opening and closing.
- To create the test cases first opening the side nav bar and then select the first address and after address selection file-manager icon arises and clicking on it. 
- After clicking on the icon file-manager opens and than after checking its status of file manager is opened or not if it is opened closing the file manager.

### CODE REVIEW

File: src/bitmessagekivy/tests/test_filemanager.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        # Click to open Address Dropdown
+        self.assertExists('//NavigationItem[0][@text=\"dropdown_nav_item\"]', timeout=2)
+        self.assertExists(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"]', timeout=1
+        )
+        is_open = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@is_open]', "is_open")
+        self.assertEqual(is_open, False)
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_filemanager.py:23]: following JSON syntax, i.e. lowercase `false`, may fix it

File: src/bitmessagekivy/tests/test_filemanager.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if self.cli.wait(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+                '/IdentitySpinner[@name=\"identity_dropdown\"][@state=\"normal\"]', timeout=2):
+            self.cli.sleep(0.2)
+        # Check the state of dropdown.
+        is_open = self.cli.getattr(
+            '//NavigationItem[0][@text=\"dropdown_nav_item\"]'
+            '/IdentitySpinner[@name=\"identity_dropdown\"][@is_open]', 'is_open'
+        )
+        self.assertEqual(is_open, True)
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_filemanager.py:41]: `true`

File: src/bitmessagekivy/tests/test_filemanager.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        self.assertExists(
+            '//ContentNavigationDrawer//MDIconButton[1][@icon=\"file-image\"]',
+            timeout=5
+        )
+        # Clicking on file manager icon
+        self.cli.wait_click(
+            '//ContentNavigationDrawer//MDIconButton[1][@icon=\"file-image\"]',
+            timeout=5)
+        # Checking the state of file manager is it open or not
+        if self.assertTrue(self.cli.execute('app.file_manager_open')):
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_filemanager.py:68]: shouldn't have any `if`s

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/bitmessagekivy/tests/test_filemanager.py (hunk 1 above)
Comment (osamacis) [reply src/bitmessagekivy/tests/test_filemanager.py:23]: Added assertExist with json syntax `false`

File: src/bitmessagekivy/tests/test_filemanager.py (hunk 2 above)
Comment (osamacis) [reply src/bitmessagekivy/tests/test_filemanager.py:41]: Added assertExist with json syntax `true`

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**